app/controllers/users/passwords_controller.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Users::PasswordsController < Devise::PasswordsController
+
+end
```

Comment:
Do we need this file if we are not changing any default actions of Devise?

Reply:
I think it would be better we have it. If we implement some thing in future, it could help others to know about the controller. 🤔

Reply:
Ehm, yeah. Makes sense. But we can just do `rails g devise:controllers -c=passwords` when we really require it, right? I am okay with either 😉